CHANGELOG.md

```diff
@@ -1,8 +1,11 @@
 ## Release notes
 
+### Current
+=======
```

Comment:
```suggestion
```
Not sure if you intentionally meant for this to go in but the `0.13.0 -- TBD` block is serving as the 'current' placeholder.

Reply:
yes, this is a side-effect from 'develop against master', wasn't sure of the best approach. 

updated/adjusted/conflict resolved. would be good to figure out the best approach to auto-prevent conflicts here if possible, though these are quick to resolve anyway.